packages/react-aria-components/docs/react-aria-components.mdx

```diff
@@ -0,0 +1,681 @@
+---
+
+# React Aria Components
+
+React Aria Components is a library of unstyled components built on top of the React Aria hooks. It provides a simpler way to build accessible components with custom styles, while offering the flexibility to drop down to hooks for even more customizability where needed.
+
+## What is React Aria Components?
+
+React Aria Components is a new library of unstyled components implementing [ARIA patterns](https://www.w3.org/WAI/ARIA/apg/), built on top of the existing [React Aria](index.html) hooks. It provides components for common UI patterns, with [accessibility](accessibility.html), [internationalization](internationalization.html), [interactions](interactions.html), and behavior built in, allowing you to focus on your unique design and styling rather than re-building these challenging aspects. React Aria has been meticulously tested across a wide variety of devices, interaction modalities, and assistive technologies to ensure the best experience possible for all users.
+
+Compared with the React Aria hooks, the components provide a default DOM structure and styling API, and abstract away the glue code necessary to connect the hooks together. The components and hooks also work together, allowing them to be mixed and matched depending on the level of customization required. Eventually, React Aria Components will be recommended as a starting point when building a new component, dropping down to hooks only when additional flexibility is needed. Both hooks and components will continue to be maintained and developed going forward.
+
+## Status
+
+React Aria Components is currently in **alpha**. This means APIs will likely change in future updates as we discover the best ways to use it, and there are some known bugs and limitations. That said, it is based on a solid and battle-tested foundation in React Aria, and we would love for you to try it out and give us feedback! This will directly help us shape the APIs and make it the best library it can be. Please report issues and feature requests [on GitHub](https://github.com/adobe/react-spectrum/issues).
+
+## Installation
+
+React Aria Components can be installed using a package manager like [npm](https://docs.npmjs.com/cli/npm) or
+[yarn](https://classic.yarnpkg.com/lang/en/).
+
+```
+yarn add react-aria-components
+```
+
+All components are available in this one package for ease of dependency management.
+
+## Implementing a component
+
+Once installed, you can import and render the components you need. Each component may include several parts, as documented on the corresponding component page. The API is designed around composition, where each component generally has a 1:1 relationship with a single DOM element. This makes it easy to style every element, and control the layout and DOM order as needed to implement your design.
+
+This example renders a custom [Select](Select.html).
```

Comment:
Do we ever have to worry about URLs collision between info pages and component pages? Should react-aria components be `react-aria/components/NAME.html`? The hooks will always have `use` in front so they should be safe. For example, we have VisuallyHidden as a page for useVisuallyHidden, so we can't have a component named that (not that I expect us to create this).